### PR TITLE
unofficial-superset-snowflake

### PR DESCRIPTION
**Pull request by MiguelElGallo**
Adding a new repo for a Helm chart, for superset connected to Snowflake.

### CODE REVIEW

**Comment by MiguelElGallo** [comment]
Hello, can you please approve this request ?

**Comment by mattfarina** [comment]
The [Artifact Hub](https://artifacthub.io/) is another place you can list your repos. To do that, you login (or create an account), create an organization, and list your repository. You can link it to a user or organization.

File: config/repo-values.yaml [hunk 1]
```diff
@@ -502,3 +502,5 @@ sync:
       url: https://charts.streamnative.io/
     - name: kured
       url: https://weaveworks.github.io/kured
+    - name: unofficial-superset-snowflake
```
Comment (mattfarina) [inline config/repo-values.yaml:505]: The name can only be 21 characters long. This one is 29. Can you please use a shorter name.

Comment (scottrigby) [reply config/repo-values.yaml:505]: Cross-referencing https://github.com/helm/hub/issues/362

**Comment by mattfarina** [comment]
The Helm Hub has been superseded by the Artifact Hub. Chart repositories should now be listed there. Closing this PR.

**Pull request closed without merging**